android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -548,15 +541,6 @@ public void onError(@NonNull String message) {
         break;
       }
       case "symbol#add": {
```

Comment:
```suggestion
      case "symbols#addAll": {
```
Could we please keep the "symbol#addAll" or possibly even better: "symbols#addAll"?
This also makes it clear that adding symbols is currently implemented differently than adding circles and lines (which we might want to update at a later time to support batch editing, as well)